data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/configuration/SqsOptions.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+
+import java.time.Duration;
+
+public class SqsOptions {
+    private final int DEFAULT_MAXIMUM_MESSAGES = 10;
+    private final int DEFAULT_VISIBILITY_TIMEOUT_SECONDS = 30;
```

Comment:
Your choice here: You could make these defaults actual `Duration` objects. Since they will be static, you won't need to re-create the `Duration` in each field. I also think it is cleaner since the code only deals with `Duration`.

e.g.

```
private static final Duration DEFAULT_VISIBILITY_TIMEOUT = Duration.ofSeconds(20);
```

Reply:
I like this suggestion.